docs/usecases/extending_data_ecosystem.rst

```diff
@@ -0,0 +1,151 @@
+endpoints that can be private or public. Therefore, when users download annexed files,
+Globus communicates with git-annex to provide access to files content. Given this
+functionality, we can say that Globus works as a data store for git-annex, or in
+other words it is configured to work as a special remote for git-annex. This is
+possible via the git-annex backend interface implementation for Globus
+called `git-annex-globus-remote <https://github.com/CONP-PCNO/git-annex-remote-globus>`_.
+
+To sum up, CONP makes available a variety of datasets and provides them to researchers
+as Datalad datasets, given the advantageous Datalad functionalities. In this way,
+researchers have access to a wide research data ecosystem without need of data replication
+thanks to the ability of git-annex and Datalad to interface with special remote locations
+across the web and request access to data !
+
+
+Globus as git-annex data store
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+A remote data store exists thanks to git-annex (which DataLad builds upon):
+Git-annex uses a key-value pair to reference files. In the git-annex object tree,
+large files in datasets are stored as values while the key is generated from their
+contents and is checked into Git. The key is used to reference the location of the value
+in the object tree. The object tree (or keystore) with the data contents can
+be located anywhere – its location only needs to be encoded using a special remote.
+Therefore, thanks to the git-annex-globus-remote interface, Globus.org provides
+git-annex with locations information to retrieve values and access files content
+with the corresponding keys. To ultimately enable end users’ access to data,
+git annex registers Globus locations assigning them to Globus-specific URLs,
+such as `globus://datadet_id/path/to/file`, where each Globus URL is associated
+with a the key corresponding to the given file. The use of a Globus URL protocol
+is a fictitious mean to assign each file of the dataset a unique location and
+source and therefore, it is a wrapper for additional validation that is performed
+by the git-annex-globus-remote to check on the actual presence of the file within
+the Globus transfer file ecosystem. In other words, the ‘Globus URL’ is simply an
+alias of an existing file located on the web and specifically available in Globus.org.
+Registration of Globus URLs in git-annex is among the configuration procedures
+carried out on an administrative, system-wide level, and users will only deal
+with direct easy access of desired files.
+
+Therefore, Globus is configured to receive request for data access from the git-annex
+side and to respond back if data is available. Currently, git-annex-globus-remote
+only supports data download operations but it could potentially be useful for additional
+functionalities. When the globus-remote get initialized for the first time, the user
+has to authenticate to Globus.org using ORCHiD, Gmail or a specific Globus accounts:
```

Comment:
I would use `ORCID` not `ORCiD`.  An `i` in ORCID with a dot is used only in the logo on top of full sized capital I.  They use ORCID elsewhere